### DISCUSSION TITLE
fuse mount performance slow

### DISCUSSION THREAD

**Discussion by paochiang** [seaweedfs/seaweedfs#1979] [Q&A]
I found that write performance through fuse mount is very poor.

For example:
replication mode is `002`

After `weed mount`, use command `cp -r /root/my-app-master/* /mnt/swfs/for` copy a folder to seaweedfs, It took more than 10s(Test multiple times，about10s ~ 20s). 

copied folder: [my-app-master.zip](https://github.com/paochiang/data/blob/main/my-app-master.zip) , about 150KB and 130 files after decompressed .

I am not sure whether this performance is in line with expectations, I personally feel that it is very slow.  

Is there any suggestion for folder operations？ (does not use filer.copy)   

Thanks!

**Comment by chrislusf** [comment]
I used replication 000 and tested locally. It can finish in 0.7 seconds.

**Reply by paochiang** [reply]
OK, let me check again

**Reply by paochiang** [reply]
I did more tests in a clean k8s cluster, if filer store is leveldb2,  It takes 1s. But if  if filer store is mysql,  it takes 6s.   So now it seems like that mysql affects performance,  and the impact is very serious. I need to further locate the problem.

**Reply by paochiang** [reply]
but if i use`filer.copy`,  mysql as  filer store  has no effect, it take about 1s.
